flytekit/remote/remote.py

```diff
@@ -566,14 +620,13 @@ def execute(
             first before executing.
         :param name: execute entity using this name. If not None, use this value instead of ``entity.name``
         :param version: execute entity using this version. If None, uses auto-generated value.
-        :param execution_name: name of the execution
+        :param execution_name: name of the execution. If None, uses auto-generated value.
         :param wait: if True, waits for execution to complete
-        :returns: :class:`~flytekit.remote.workflow_execution.FlyteWorkflowExecution`
 
         .. note::
 
-            The ``project``, ``domain``, ``name``. and ``version`` arguments do not apply to ``FlyteTask``,
-            ``FlyteLaunchPlan``, and ``FlyteWorkflow`` objects.
+            The ``project``, ``domain``, ``name`` and ``version`` arguments do not apply to ``FlyteTask``,
```

Comment:
does this mean users can never override the project or domain that an execution is launched under?